rllib/policy/view_requirement.py

```diff
@@ -29,7 +29,7 @@ class ViewRequirement:
     def __init__(self,
                  data_col: Optional[str] = None,
                  space: gym.Space = None,
-                 shift: Union[int, List[int]] = 0,
+                 data_rel_pos: Union[int, List[int]] = 0,
```

Comment:
Why not keep it as shift? It seems to be intuitive

Reply:
I liked `shift`, too. The problem is, there will also be an `abs_pos` soon (see attention net PRs). So I wanted to distinguish between these two concepts.